doc/classes/TransformContainer.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TransformContainer" inherits="Container" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Applies transform (offset, rotation, scale) to child controls.
+	</brief_description>
+	<description>
+		[TransformContainer] will transform all its child [Control] nodes according to the specified transform parameters. The control is first stretched to full rect (like in [MarginContainer]) and then the transform is applied on top. You can freely offset, scale and rotate the node, making it useful for animations.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="transform_offset" type="Vector2" setter="set_transform_offset" getter="get_transform_offset" default="Vector2(0, 0)">
+			Position offset applied to child controls. If [member transform_offset_relative_to_size] is [code]true[/code], the offset is relative to the node's [member Control.size].
```

Comment:
```suggestion
			Position offset applied to this container's child controls. If [member transform_offset_relative_to_size] is [code]true[/code], the offset is relative to the container's [member Control.size].
```

Not sure whether to say "this container" or "this node". This sentiment propagates to the other suggestions